internal/generated/snippets/go.mod

```diff
@@ -24,100 +24,100 @@ replace cloud.google.com/go/storage => ../../../storage
 
 require (
 	cloud.google.com/go v0.93.3
-	cloud.google.com/go/accessapproval v0.1.0
-	cloud.google.com/go/aiplatform v0.1.0
-	cloud.google.com/go/analytics v0.1.0
-	cloud.google.com/go/apigateway v0.1.0
-	cloud.google.com/go/apigeeconnect v0.1.0
-	cloud.google.com/go/appengine v0.1.0
-	cloud.google.com/go/area120 v0.1.0
-	cloud.google.com/go/artifactregistry v0.1.0
-	cloud.google.com/go/asset v0.1.0
-	cloud.google.com/go/assuredworkloads v0.1.0
-	cloud.google.com/go/automl v0.1.0
-	cloud.google.com/go/bigquery v0.90.0
-	cloud.google.com/go/billing v0.1.0
-	cloud.google.com/go/binaryauthorization v0.1.0
-	cloud.google.com/go/channel v0.1.0
-	cloud.google.com/go/cloudbuild v0.1.0
-	cloud.google.com/go/clouddms v0.1.0
-	cloud.google.com/go/cloudtasks v0.1.0
-	cloud.google.com/go/container v0.1.0
-	cloud.google.com/go/containeranalysis v0.1.0
-	cloud.google.com/go/datacatalog v0.1.0
-	cloud.google.com/go/dataflow v0.1.0
-	cloud.google.com/go/datafusion v0.1.0
-	cloud.google.com/go/datalabeling v0.1.0
-	cloud.google.com/go/dataproc v0.1.0
-	cloud.google.com/go/dataqna v0.1.0
-	cloud.google.com/go/datastore v0.90.0
-	cloud.google.com/go/datastream v0.1.0
-	cloud.google.com/go/dialogflow v0.1.0
-	cloud.google.com/go/dlp v0.1.0
-	cloud.google.com/go/documentai v0.1.0
-	cloud.google.com/go/domains v0.1.0
-	cloud.google.com/go/errorreporting v0.1.0
-	cloud.google.com/go/essentialcontacts v0.1.0
-	cloud.google.com/go/eventarc v0.1.0
-	cloud.google.com/go/firestore v0.90.0
-	cloud.google.com/go/functions v0.1.0
-	cloud.google.com/go/gaming v0.1.0
-	cloud.google.com/go/gkeconnect v0.1.0
-	cloud.google.com/go/gkehub v0.1.0
-	cloud.google.com/go/gsuiteaddons v0.1.0
-	cloud.google.com/go/iap v0.1.0
-	cloud.google.com/go/iot v0.1.0
-	cloud.google.com/go/kms v0.1.0
-	cloud.google.com/go/language v0.1.0
-	cloud.google.com/go/lifesciences v0.1.0
-	cloud.google.com/go/logging v0.90.0
-	cloud.google.com/go/managedidentities v0.1.0
-	cloud.google.com/go/mediatranslation v0.1.0
-	cloud.google.com/go/memcache v0.1.0
-	cloud.google.com/go/metastore v0.1.0
-	cloud.google.com/go/monitoring v0.1.0
-	cloud.google.com/go/networkconnectivity v0.1.0
-	cloud.google.com/go/networkmanagement v0.1.0
-	cloud.google.com/go/notebooks v0.1.0
-	cloud.google.com/go/orgpolicy v0.1.0
-	cloud.google.com/go/osconfig v0.1.0
-	cloud.google.com/go/oslogin v0.1.0
-	cloud.google.com/go/phishingprotection v0.1.0
-	cloud.google.com/go/policytroubleshooter v0.1.0
-	cloud.google.com/go/privatecatalog v0.1.0
+	cloud.google.com/go/accessapproval v0.93.3
+	cloud.google.com/go/aiplatform v0.93.3
+	cloud.google.com/go/analytics v0.93.3
+	cloud.google.com/go/apigateway v0.93.3
+	cloud.google.com/go/apigeeconnect v0.93.3
+	cloud.google.com/go/appengine v0.93.3
+	cloud.google.com/go/area120 v0.93.3
+	cloud.google.com/go/artifactregistry v0.93.3
+	cloud.google.com/go/asset v0.93.3
+	cloud.google.com/go/assuredworkloads v0.93.3
+	cloud.google.com/go/automl v0.93.3
+	cloud.google.com/go/bigquery v0.93.3
```

Comment:
This seems...wrong.

Reply:
Yeah, I believe there is a bug in renovate /w submodules that are at a lower version than their root module. It does not affect us right now because it is only happening in a file where we replace all the deps anyways, but this is not great for people who are consuming our api updates.